packages/validator/src/services/validatorStore.ts

```diff
@@ -228,6 +232,10 @@ export class ValidatorStore {
     );
   }
 
+  getValidatorKeys(): PubkeyHex[] {
```

Comment:
this function seems to do the same as `votingPubkeys`

https://github.com/ChainSafe/lodestar/blob/df1ef97e9050a588b74f21bfae314d675caea288/packages/validator/src/services/validatorStore.ts#L312-L314

if we want to keep it I would at least improve naming consistency and use the term pubkey

```suggestion
  getValidatorPubkeys(): PubkeyHex[] {
```

Reply:
Good catch, I will just remove the function instead.